static/demo/demo.css

```diff
@@ -41,3 +41,7 @@ html, body {
 #playground.hidden {
   display: none;
 }
+
+.ffwdme-debug-wrapper>button {
+    margin-top: 100px;
+}
```

Comment:
Please move this here: https://github.com/ffwdme/ffwdme.js/blob/master/src/debug/debug.scss#L35

Or remove :)